spring-integration-core/src/main/java/org/springframework/integration/support/management/graph/LinkNode.java

```diff
@@ -50,7 +50,7 @@ public Type getType() {
 	}
 
 	public enum Type {
-		input, output, error, discard
+		input, output, error, discard, route
```

Comment:
>  I am not sure what documentation you are looking for.

The `route` is a new link type. See `graph.adoc`:
```
For the convenience and to allow to determine a link purpose, the model is supplied with the `type` attribute.
The possible types are:

- _input_ - identify the direction from `MessageChannel` to the endpoint; `inputChannel` or `requestChannel` property;
```

Also a new `properties` has right to appear there as well with mentioning that we provide the `primaryExpression` there for better visualization. 
WDYT?